### ISSUE TITLE
Module build failing (v1.1.9)

### ISSUE THREAD

**Issue by echeyne** [angelnikolov/ngx-cacheable#37]
I just upgraded version of 1.1.9 of `ngx-cacheable` and when I try and compile my app I get the following error:

```ERROR in ./node_modules/ngx-cacheable/index.ts
Module build failed (from ./node_modules/@ngtools/webpack/src/index.js):
Error: C:\Users\Emily Cheyne\source\repos\EmilyEMRVS\EmilyEMRVS\ClientApp\node_modules\ngx-cacheable\index.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
The missing file seems to be part of a third party library. TS files in published libraries are often a sign of a badly packaged library. Please open an issue in the library repository to alert its author and ask them to package the library using the Angular Package Format (https://goo.gl/jB3GVv).
    at AngularCompilerPlugin.getCompiledFile (C:\Users\Emily Cheyne\source\repos\EmilyEMRVS\EmilyEMRVS\ClientApp\node_modules\@ngtools\webpack\src\angular_compiler_plugin.js:752:23)
    at plugin.done.then (C:\Users\Emily Cheyne\source\repos\EmilyEMRVS\EmilyEMRVS\ClientApp\node_modules\@ngtools\webpack\src\loader.js:41:31)
    at process._tickCallback (internal/process/next_tick.js:68:7)```

**Comment by angelnikolov** [comment]
@echeyne Should be fixed in 1.2.0. Would you please try again and close the issue if it works :)?
Thank you!

**Comment by echeyne** [comment]
Fixed, thanks for such a fast response :)
